app/models/has_corporate_information_pages.rb

```diff
@@ -31,10 +31,10 @@ def published_corporate_information_pages
   end
 
   def about_us
-    @about_us ||= corporate_information_pages.published.for_slug("about")
+    corporate_information_pages.published.for_slug("about")
   end
 
   def draft_about_us
-    @draft_about_us ||= corporate_information_pages.draft.for_slug("about")
+    corporate_information_pages.draft.for_slug("about")
```

Comment:
Do the `summary` and the `body` (which are defined based upon the `about_us`) need to be draft-aware as well?

Reply:
I don't believe `body` does, in fact I'm not 100% sure it's used at all. But you are correct about the `summary` that's still broken, more work to do 🙃

Reply:
This is fixed now Cristina :) I've not touched the potentially unused `body` method, as I think refactoring that is out of scope of this card.

Reply:
Thanks!

I don't suppose there's a way to make the entity that `has_corporate_information_pages` aware of `state`, such as `about_us` and `draft_about_us` become one method:
```
def about_us
  corporate_information_pages.send(state.to_sym).for_slug("about")
end
```
or similar?